src/gui/sdl_mapper.cpp

```diff
@@ -1228,7 +1228,8 @@ class Typer {
 			m_pace_ms = pace_ms;
 			m_stop_requested = false;
 			m_instance = std::thread(&Typer::Callback, this);
-		}
+		        set_thread_name(m_instance, "dosbox-autotype");
```

Comment:
Whitespace.

Reply:
Interesting - this is somehow confusing clang-format.
Even when I manually correct it with tabs, clang-format switches it back to `<tab><tab> + 8-spaces`.
(manually fixing - thanks for catching it!)